### PR TITLE
DAOS-7821 cq: Resolve python flake warnings.

### PR DESCRIPTION
**Pull request by ashleypittman**
Fix all instances of W504, E226 and E265.  Fix other python
warnings in files being touched anyway, and yaml warnings when
the python code was being modified.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: src/tests/ftest/util/performance_test_base.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from exception_utils import CommandFailure
 
 
+# TODO dmg system query as non-json to reduce log clutter # pylint: disable=fixme
```
Comment (daltonbohning) [inline src/tests/ftest/util/performance_test_base.py:20]: This comment has since been removed entirely in master
```suggestion
```

**Review by daltonbohning — CHANGES_REQUESTED** [review]
Just one comment to be removed - likely the result of a merge conflict

File: src/tests/ftest/util/performance_test_base.py (hunk 1 above)
Comment (ashleypittman) [reply src/tests/ftest/util/performance_test_base.py:20]: Oh yeah, bad merge.  I'll fix.

**Review by daltonbohning — APPROVED** [review]

**Review by mark-olson — COMMENTED** [review]
I looked at it and only had 2 questions.
I will let @JohnMalmberg and @brianjmurrell give the approval.

File: src/tests/ftest/aggregation/punching.py [hunk 2]
```diff
@@ -89,18 +85,16 @@ def test_aggregation_punching(self):
...
                 break
-            else:
-                self.log.info("Space is not reclaimed yet !")
-                self.log.info("Sleeping for 30 seconds")
-                time.sleep(30)
-                counter += 1
+            self.log.info("Space is not reclaimed yet !")
+            self.log.info("Sleeping for 30 seconds")
+            time.sleep(30)
+            counter += 1
```
[the remark this answers is not in the document]
Comment (ashleypittman) [reply src/tests/ftest/aggregation/punching.py:91]: No, the if block breaks the loop so this code is logically the same.

https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-break.html

File: src/tests/ftest/util/soak_test_base.py [hunk 3]
```diff
@@ -305,7 +304,7 @@ def job_setup(self, jobs, pool):
                 # nodesperjob = -1 indicates to use all nodes in client hostlist
```
[the remark this answers is not in the document]
Comment (ashleypittman) [reply src/tests/ftest/util/soak_test_base.py:304]: Yes probably, but I'm only making the easy changes in order to enable linting checks.

**Review by daltonbohning — DISMISSED** [review]

**Review by shimizukko — APPROVED** [review]

**Review by jolivier23 — DISMISSED** [review]
Looks fine to me but now has conflicts to resolve

**Review by daltonbohning — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Pull request merged**